controller/rancher/rancher.go

```diff
@@ -156,23 +163,53 @@ func (lbc *LoadBalancerController) BuildConfigFromMetadata(lbName, envUUID, self
 		}
 	}
 	frontendsMap := map[string]*config.FrontendService{}
-	// fetch certificates
+
+	// fetch certificates either from mounted certDir or from the cattle
 	certs := []*config.Certificate{}
-	for _, certName := range lbMeta.Certs {
-		cert, err := lbc.CertFetcher.FetchCertificate(certName)
+	var defaultCert *config.Certificate
```

Comment:
@prachidamle  labels can't be changed for the running container; only via upgrade (the container will be recreated). So we can determine default Cert dir and cert dir at the very start, and put them to CertFetcher on its init stage. These values won't change.